Class/C-Array-Methods/exercise4.js

```diff
@@ -4,12 +4,18 @@
 let numbersList = [5, 3, 15, 8, 24, 31, 12];
 
 function doubleValue(x){
-    return x * 2;
+    //return x * 2;
 }
 
 let doubledNumbers = [];
 

```

Comment:
You can do everything for this exercise by editing line 10 so that `doubledNumbers` is set to a value returned by by `numbersList.map()`.